src/hypothesis/core.py

```diff
@@ -607,10 +607,11 @@ def wrapped_test(*arguments, **kwargs):
 
             random = get_random_for_wrapped_test(test, wrapped_test)
 
-            test_runner, search_strategy = process_arguments_to_given(
+            _processed_args = process_arguments_to_given(
                 wrapped_test, arguments, kwargs, generator_arguments,
                 generator_kwargs, argspec, test, settings
             )
+            arguments, kwargs, test_runner, search_strategy = _processed_args
```

Comment:
FWIW I don't find splitting out the variable here clearer than just doing the assignment all in one go (I guess it's probably for line length reasons?) and think the leading underscore looks a bit odd for a local variable.

But I also don't really care that much and am happy to leave it as is.